db.go

```diff
@@ -60,13 +60,42 @@ func Create(path string) (*DB, error) {
 // Open opens the database at the location path using mode. Caller is responsible
 // for closing the database when done.
 func Open(path string, mode DBMode) (*DB, error) {
-	cpath := C.CString(path)
-	defer C.free(unsafe.Pointer(cpath))
+	config := ""
```

Comment:
that's probably the wrong default, I'd pass nil

Reply:
oh wait, if that's what upstream is doing (the C lib) by all means ignore me.

We should do the same